packages/terra-theme-context/src/ThemeContext.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const ThemeProviderContext = React.createContext({});
+
```

Comment:
One thing I've been trying to do is define/export the shape of a context's value along with the Context creation: https://github.com/cerner/terra-application/blob/master/src/application-loading-overlay/ApplicationLoadingOverlayContext.jsx#L6

I'm not sure how useful that shape is now that the useContext hook has killed most usage of the HOC prop injection patterns, but it might come in handy? It'd be cooler if the props-table loader could parse an arbitrary file so we could ditch the manual tables in the docs.

Reply:
added below